### DISCUSSION TITLE
Textarea field with a variable word inside?

### DISCUSSION THREAD

**Discussion by notflip** [statamic/cms#7020] [Q&A] [answered]
I have a textarea field, in which my client can add text, inside this text there will be an amount of days (until the event), for example

`We're counting down! {{ amountOfDays }} until the event happens!`

This will be calculated using the current date, minus a global set date, is this possible using Statamic?

**Comment by ryanmitchell** [comment, marked as answer]
Yes, you select the following field in the config:

<img width="454" alt="Screenshot 2022-11-08 at 09 29 22" src="https://user-images.githubusercontent.com/51899/200527676-b346d1b8-e66d-4499-b542-0f2647a21156.png">

Then you can specify the days as a variable in your template or in a view model

**Reply by notflip** [reply]
Thanks @ryanmitchell I managed to output that variable! I'm wondering though, the output is from another date field, I want to compute the amount of days between then and now, I looked at the computed properties documentation but it seems that's only for Users and Collections, do I need to use JS to read, process and overwrite the data in that div? Or can statamic tell me the difference in days between two dates?
